packages/@apphosting/adapter-angular/src/utils.ts

```diff
@@ -183,7 +183,9 @@ async function generateBundleYaml(
       headers: [],
       redirects: [],
       rewrites: [],
-      runCommand: `node ${normalize(relative(cwd, outputPathOptions.serverFilePath))}`,
+      runCommand: `SSR_PORT=$PORT node ${normalize(
```

Comment:
Can you add a comment that note what version this fix is needed for, we many want to have a separate versioned adapter in the future

Reply:
added a comment below

Reply:
My bad, just added!